builds/dumbo/scripts/shared/userService.js

```diff
@@ -7,8 +7,9 @@ angular.module('dumboApp')
     setLoggedIn : function(userEmail){
       $cookies.put('email', userEmail);
     },
-    isLoggedin : function() {
-      return ($cookies.get('email') != undefined);
+    isLoggedIn : function() {
+      var c = $cookies.get('email');
+      return (c != null && c != "undefined");
```

Comment:
Good catch. There should be a typeof() around c. Typeof() returns a string, which in that case may be the string 'undefined' if the object is indeed undefined. I'll patch in a fix soon.